__test__/api/comment-get.test.js

```diff
@@ -85,3 +85,29 @@ describeIfEndpoint(
     });
   }
 );
+
+describeIfEndpoint(
```

Comment:
replace `describeIfEndpoint(string, string, string)` with the default `describe(string)`
The helper function was required previously when some endpoints were not implemented yet, but no longer required